File: Android /Android/Separate Android /Android/JMBA_Olo_Android_SDK/app/src/main/java/com/fishbowl/apps/olo/Models/OloBasketProductBatchResult.java
package com.fishbowl.apps.olo.Models;

import java.util.ArrayList;

/**
 * Created by Nauman Afzaal on 05/05/15.
 */
public class OloBasketProductBatchResult
{
    private OloBasket basket;
    private ArrayList<OloBasketProductBatchError> errors;

    public OloBasket getBasket()
    {
        return basket;
    }

    public ArrayList<OloBasketProductBatchError> getErrors()
    {
        return errors;
    }
}

File: Android /Android/25:Jan:2017/FishbowlMobileApps/AndroidSDK/FishbowlMobileApps-Jamba-Android-v2.4.2/JMBA_Olo_iOS_SDK/Olo/OloBasketProductBatchResult.swift
//
//  OloBasketProductBatchResult.swift
//  Olo
//
//  Created by Taha Samad on 05/05/2015.
//  Copyright (c) 2015 Hathway, Inc. All rights reserved.
//

import Foundation
import SwiftyJSON

open class OloBasketProductBatchResult {
    
    open var basket: OloBasket
    open var errors: [OloBasketProductBatchError]
    
    public init(json: JSON) {
        basket = OloBasket(json: json["basket"])
        errors = json["errors"].arrayValue.map { item in OloBasketProductBatchError(json: item) }
    }
    
}
